src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -41,9 +49,23 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
 
           <ShowInstallShots show={show} my={2} />
 
-          <Text as="h1" variant="largeTitle" my={4}>
-            {show.name}
-          </Text>
+          <GridColumns>
+            <Column span={hasWideHeader ? [12, 8, 6] : 6} wrap={hasWideHeader}>
```

Comment:
This actually would have been much more complex to express with the notion of a `Row` component. With this boolean for wrapping, things are a lot simpler, which I find surprising/interesting.